workloads/kube-burner/common.sh

```diff
@@ -16,6 +16,16 @@ log() {
   echo -e "\033[1m$(date "+%d-%m-%YT%H:%M:%S") ${@}\033[0m"
 }
 
+collect_pprof() {
+  sleep 50
+  while [ $(oc get benchmark -n benchmark-operator kube-burner-${1}-${UUID} -o jsonpath="{.status.complete}") == "false" ]; do
+    log "-----------------------checking for new pprof files--------------------------"
+    oc rsync -n benchmark-operator $(oc get pod -n benchmark-operator | grep "kube-burner" | grep Running | awk '{print $1}'):/tmp/pprof-data $PWD/
+    sleep 60
```

Comment:
Move this to first step in this while loop and remove sleep 50 from top ?

Reply:
nope , the while condition , that is the oc get benchmark will fail since the kube-burner workload hasn't initialized yet

Reply:
Maybe use something like this, as done in 
`local timeout=$(date -d "+${POD_READY_TIMEOUT} seconds" +%s)
  until oc get benchmark -n my-ripsaw kube-burner-${1}-${UUID} --ignore-not-found -o jsonpath="{.status.state}" | grep -q Complete; do
    sleep 1
    if [[ $(date +%s) -gt ${timeout} ]]; then
      log "Timeout waiting for job to be created"
      exit 1
    fi`

instead of hard-coding the sleep time

Reply:
it's not possible to measure the time / check since that part is handled by the kube-burner binary, and it usually takes around 30sec for the kube-burner job to start till the pprof measurement begins